MonoGame.Framework/Input/IImeService.cs

```diff
@@ -0,0 +1,67 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+
+namespace Microsoft.Xna.Framework
+{
+    /// <summary>
+    /// Interface to handle text composition.
+    /// </summary>
+    public interface IImeService
+    {
+        /// <summary>
+        /// Enable the system IMM service to support composited character input.
+        /// This should be called when you expect text input from a user and you support languages
+        /// that require an IME (Input Method Editor).
+        /// </summary>
+        void StartTextInput();
+
+        /// <summary>
+        /// Stop the system IMM service.
+        /// </summary>
+        void StopTextInput();
+
+        /// <summary>
+        /// Show the Ime Candidate window renderer by the OS.
+        /// In SDL2 platform it's always true since there's no API to get candidate texts in SDL2.
+        /// In WindowsDX platform, it's possible to choose show candiate window renderer by the OS or render it yourself.
```

Comment:
```suggestion
        /// Show the Ime Candidate window rendered by the OS.
        /// Set to <c>false</c> if you want to render the IME candidate list yourself.
        /// On DesktopGL the candidate list is not exposed and this will always return <c>true</c>.
```

Reply:
Is this only about the candidate list or does the OS window also show the current composition?
We should make that clear in the docs.
If it's only the candidate list we should probably rename the property to `RenderPlatformCandidateList` or something like that.

Reply:
No, it's not just about the candidate list, the OS renders nothing about IME in SDL2, So in SDL2, we have to render both text composition and candidate list by ourself.